### DISCUSSION TITLE
Return value of Tymon\\JWTAuth\\Providers\\JWT\\Lcobucci::getBuilderFromClaims() must be an instance of Lcobucci\\JWT\\Token\\Builder, instance of Lcobucci\\JWT\\Builder returned

### DISCUSSION THREAD

**Discussion by koti42** [lcobucci/jwt#1007] [Q&A]
I'm getting the error above, I'm using this version, my php version 7.2 laravel version 5.5 Suddenly this error started to appear, how can I solve it?
        "tymon/jwt-auth": "dev-develop",

**Comment by SvenRtbg** [comment]
I know it's easy to rant about it, and you don't have an alternative in this particular combination of dependencies apparantly, but depending on branches usually is a bad idea. Then again, the software you are referring to isn't maintained in this repo, so: Wrong place to complain about as step 1. And last: You are not really sending any additional info about what you are doing. What call is failing on your side? Do you expect anyone to be familiar with a third party software utilizing this library and invest some decent amount of debugging without any hint what your code is doing?

Still you may get some help if you can fill us in about the missing details, or you can start testing which version of lcobucci/jwt is upsetting your software by requiring older versions. This lib has undergone some serious internal reconstruction and moving stuff around, so your problem doesn't sound new - still you may be out of luck when trying to use the latest version.
